src/cli/src/install/mod.rs

```diff
@@ -122,12 +122,12 @@ pub fn install_bin<P: AsRef<Path>, B: AsRef<[u8]>>(bin_path: P, bytes: B) -> Res
     let bin_path = bin_path.as_ref();
 
     // Create directories to bin_path if they do not exist
-    if let Some(parent) = bin_path.parent() {
-        std::fs::create_dir_all(&parent)?;
-    }
+    let parent = bin_path.parent()
+        .ok_or_else(|| IoError::new(ErrorKind::NotFound, "parent directory not found"))?;
+    std::fs::create_dir_all(&parent)?;
```

Comment:
How could the `parent` not already exist?

Reply:
I cannot think of an example, but `Path::parent` returns an `Option<&Path>`, so this is just how I went about dealing with it. I imagine this code path will never be taken

Reply:
> Returns [None] if the path terminates in a root or prefix.

Reply:
So perhaps if somebody executed `fluvio` in the root directory of a container or something...